build/flaws/broken-links.js

```diff
@@ -164,6 +164,11 @@ function getBrokenLinksFlaws(doc, $, { rawContent }, level) {
           "Link points to the page it's already on"
         );
       }
+    } else if (
+      hrefNormalized.endsWith("/contributors.txt") &&
+      hrefNormalized.startsWith("/")
+    ) {
+      // Do nothing. The /contributors.txt URLs are special Yari URLs.
```

Comment:
@peterbe What do you think about moving these "do nothing" cases above this long "do something" `if-else` statement? So maybe right after we create and define the `hrefNormalized` variable near the top of this loop function, we do something like the following that we can add more cases to over time?

```js
if (
      hrefNormalized.endsWith("/contributors.txt") &&
      hrefNormalized.startsWith("/")
    ) {
  return;
}
```